src/app/modules/gb-cohort-selection-module/constraint-components/gb-concept-constraint/gb-concept-constraint.component.ts

```diff
@@ -632,6 +653,16 @@ export class GbConceptConstraintComponent extends GbConstraintComponent implemen
     this.showMoreOptions = !this.showMoreOptions;
   }
 
+  onDragEnter(event) {
+    event.stopPropagation();
+    event.preventDefault();
+  }
+
+  onDragOver(event) {
+    event.stopPropagation();
+    event.preventDefault();
+  }
```

Comment:
Why these functions has to be overwritten here? They are already defined like this in the parent class (GbConstraintComponent)

Reply:
Dragging behaviour has now been fixed:
- dragging study constraints to concept or study constraints, adds the studies to the existing constraint
- otherwise, constraints are added to the parent combination constraint.